osu.Game/Database/DatabaseBackedStore.cs

```diff
@@ -2,27 +2,23 @@
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Linq.Expressions;
+using Microsoft.EntityFrameworkCore;
 using osu.Framework.Logging;
 using osu.Framework.Platform;
-using SQLite.Net;
-using SQLiteNetExtensions.Extensions;
 
 namespace osu.Game.Database
 {
     public abstract class DatabaseBackedStore
     {
         protected readonly Storage Storage;
-        protected readonly SQLiteConnection Connection;
+        protected readonly OsuDbContext Connection;
 
-        protected virtual int StoreVersion => 1;

```

Comment:
????

Reply:
As far as I understand, this property was used only for DB migrations. In EF we can migrate DB without it. I can just call `dbContext.Database.Migrate();` and pending migrations will be applied(if there are any).